chapters/functions.tex

```diff
@@ -2046,10 +2046,9 @@ \subsubsection{Simple Types}\label{simple-types}
 \end{tabular}
 \end{center}
 
-Sending string literals to FORTRAN~77 subroutines/functions is supported
-for Lapack/Blas-routines, and the strings are \textsc{nul}-terminated for
-compatibility with C. Returning strings from FORTRAN~77
-subroutines/functions is currently not supported.
+Sending string literals to FORTRAN~77 subroutines/functions is supported for Lapack/Blas-routines, and the strings are \textsc{nul}-terminated for compatibility with C.
+The strings should be UTF-8 encoded similarly as for C to be compatible with C, even though the support in FORTRAN~77 is unclear and it is not relevant for the Lapack/Blas-routines.
```

Comment:
```suggestion
The strings should be UTF-8 encoded similarly as for C to be compatible with C, even though the support in FORTRAN~77 is unclear and it is not relevant for the LAPACK/BLAS-routines.
```
Note that LAPCK no longer is FORTRAN 77, but FORTRAN 90 since v3.2.

Reply:
Right, so modern LAPACK isn't usable with Modelica.

Reply:
Exactly.